### DISCUSSION TITLE
How to disable undo shortcut?

### DISCUSSION THREAD

**Discussion by sudoexec** [nocodb/nocodb#5979] [Q&A] [answered]
The undo shortcut is very annoying for me.
I often add a row through clicking the plus button in the right-top corner, and I'll input long text content. 
I want to use ctrl+z to undo some input, but it undo some operations to other records. 

At least, I think ctrl+z shouldn't undo previous operation when  editting new row.

**Comment by dstala** [comment, marked as answer]
fixed in this PR
https://github.com/nocodb/nocodb/pull/5989
